target/linux/ath79/dts/qca9563_dlink_dir-842.dtsi

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+		led-upgrade = &power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+
+		reset {
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	// Pull up on boot - otherwise the reset button won't work
```

Comment:
Haven't seen // comments in OpenWrt DTSes so far. Do we have a preference for /* */ or are both possible?

Reply:
// is a valid comments also, I can change to /* */ if you prefer it

Reply:
@blocktrron I'm primarily interested in whether there is an OpenWrt rule for that?

Reply:
There's no preferation from the OpenWrt side. Upstream has (to my knowledge) only a preference for multi-line comments.

Reply:
Okay, thanks.
@jackcolentern Keep it.

Reply:
OK, Thanks!